buildenv/jenkins/clang_format/Dockerfile

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# Copyright IBM Corp. and others 2025
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+###############################################################################
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN set -eux; \
+    microdnf install -y \
+       wget tar xz ncurses-compat-libs \
```

Comment:
`ncurses-compat-libs` is needed for the `clang-format` binary (specifically `libtinfo.so.5`). Unfortunately, the binary will output the following message
```
clang-format: /lib64/libtinfo.so.5: no version information available (required by clang-format)
```
This is because whoever built libtinfo didn't do so with symbol versioning enabled (as per this [stack overflow answer](https://stackoverflow.com/a/63730734/8729545)). If the output is distracting we can always update `clangFormatCheck.sh` with `2>/dev/null`, but that's probably going to hide other errors in the future.